src/reducers/entities/files.js

```diff
@@ -66,7 +66,21 @@ function fileIdsByPostId(state = {}, action) {
     }
 }
 
+function filePublicLink(state = {}, action) {
```

Comment:
Should the default state be `''`?

Reply:
It should be `{}` since `action.data` is `{ "link": ... }`. So, I make a mistake in returning `''` on `LOGOUT_SUCCESS`, which should be `{}` 😅